tests/unit/services/sl-modal-test.js

```diff
@@ -0,0 +1,66 @@
+import Ember from 'ember';
+import { moduleFor, test } from 'ember-qunit';
+import ModalService from 'sl-ember-components/services/sl-modal';
+
+let service = ModalService.create();
+
+let MockModal = Ember.Object.extend({
```

Comment:
Looking at how this is being used in each test this should be able to be replaced with just a plain object.